lib/tasks/gettext.rake

```diff
@@ -52,7 +52,9 @@ end
 desc 'Invokes msgmerge to update pot/po files'
 task :updatepo do
   backup_po_files
+  system("mv #{POT_FILE} #{POT_FILE}.bak")
```

Comment:
I don't get it, this is what `backup_po_files` does no?

Reply:
To po files, yes, not to the pot file